File: Examples/CSharp/Presentations/ConvertToPDF.cs
using System.IO;
using Aspose.Slides;
using Aspose.Slides.Export;

namespace Aspose.Slides.Examples.CSharp.Presentations
{
    public class ConvertToPDF
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Presentations();

            // Instantiate a Presentation object that represents a presentation file
            Presentation presentation = new Presentation(dataDir + "ConvertToPDF.pptx");
            
            //Save the presentation to PDF with default options
            presentation.Save(dataDir + "output.pdf", SaveFormat.Pdf);
                        
        }
    }
}

File: Examples/VisualBasic/Presentations/ConvertToPDF.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Slides

Namespace Aspose.Slides.Examples.VisualBasic.Presentations
    Public Class ConvertToPDF
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Presentations()

            'Instantiate a Presentation object that represents a presentation file
            Dim pres As New Presentation(dataDir & "ConvertToPDF.pptx")

            'Save the presentation to PDF with default options
            pres.Save(dataDir & "PDFUsingDefaultOptions.pdf", Aspose.Slides.Export.SaveFormat.Pdf)

        End Sub
    End Class
End Namespace